CHANGELOG.md

```diff
@@ -20,10 +22,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Default Petastorm reader pool is changed from `process` to `thread` for lower memory usage. ([#3665](https://github.com/horovod/horovod/pull/3665))
+
 ### Deprecated
 
 ### Removed
 
+- `shuffle_buffer_size` field is removed. Global shuffle is enabled by default. Use `shuffle` to enable shuffle or not. ([#3665](https://github.com/horovod/horovod/pull/3665))
```

Comment:
```suggestion
- Removed field `shuffle_buffer_size` from `EstimatorParams`. Global shuffle is enabled by default. Use `shuffle` to enable shuffle or not. ([#3665](https://github.com/horovod/horovod/pull/3665))
```

Reply:
I will move this part from `Removed` to `Deprecated` as discussion below.